features/FEATURE_BLE/source/generic/GenericSecurityManager.cpp

```diff
@@ -772,6 +791,17 @@ void GenericSecurityManager::on_connected(
     if (dist_flags) {
         *static_cast<pal::SecurityDistributionFlags_t*>(cb) = *dist_flags;
     }
+
+    const bool signing = cb->signing_override_default ?
```

Comment:
Could you place the symbol of the conditional operator consistently: `?` is at the end of a line while `:` is at the beginning of another.

Reply:
I don't think that's inconsistent, they mean different things

Reply:
Could you explain what's different between the two ?

Reply:
They are different symbols. ? refers to the thing on the same line so should be on the same line
the : is like a && or || and refers to the thing on the same line. I put the : on the next line for the same reason I do
```
if (condition
    && condition
    || condition) {
}
```

Reply:
I can see your point. Unfortunately we do not have rules indicating how binary expression should be split across multiple lines. Would be interesting to have one.